tasks/main.yml

```diff
@@ -1,14 +1,21 @@
 ---
+- name: Gather OS specific variables.
+  include_vars: "{{ item }}"
+  with_first_found:
+      - "vars/{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
+      - "vars/{{ ansible_distribution }}.yml"
+      - "defaults/main.yml"
```

Comment:
I wanted to use the `first_found` lookup plugin with the parameter `errors` set to ignore, but the travis build fails. Until then the using a loop and falling back to the `defaults/main.yml` as last resort.